manifests/JetBrains/Toolbox/1.17.6856.yaml

```
@@ -0,0 +1,17 @@
+Id: JetBrains.Toolbox
+Version: 1.17.6856
+Name: JetBrains Toolbox
+Publisher: JetBrains
+License: Copyright (c) JetBrains s.r.o.
+AppMoniker: toolbox
+Tags: jetbrains, toolbox, download
+Description: A suite of professional development tools for a range of languages and technologies
+Homepage: https://www.jetbrains.com/toolbox-app/
+Switches:
+  Silent: /S
+  InstallLocation: /D=<INSTALLPATH>
```

It is possible that quotes are required here.
```suggestion
  InstallLocation: /D="<INSTALLPATH>"
```

Other approved manifest samples I found in winget use quotes around `<INSTALLPATH>`.